### PR TITLE
Update php-cs-fixer

### PR DESCRIPTION
**Pull request by iamflorencejay**
Updated the CDN installer to the recommended version which is on https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/doc/installation.rst

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ LABEL "repository"="http://github.com/oskarstark/php-cs-fixer-ga"
 LABEL "homepage"="http://github.com/actions"
 LABEL "maintainer"="Oskar Stark <oskarstark@googlemail.com>"
 
-RUN wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v3.0.0/php-cs-fixer.phar -O php-cs-fixer \
+RUN wget https://cs.symfony.com/download/php-cs-fixer-v2.phar -O php-cs-fixer \
```
Comment (OskarStark) [inline Dockerfile:12]:
```suggestion
RUN wget https://cs.symfony.com/download/php-cs-fixer-v3.phar -O php-cs-fixer \
```

?

**Comment by OskarStark** [comment]
What is you use case?

**Comment by iamflorencejay** [comment]
@OskarStark For some reason, my github workflow throwing some false php-cs-fixer errors because that file is not on the repo.

Screenshot_2021-06-12_10-42-24

**Comment by OskarStark** [comment]
But do you want to use PHP-CS-Fixer 3 they changed some things, check their changelog.

Otherwise you should be able to use: oskarstark/php-cs-fixer-ga@2.19.0

**Pull request closed without merging**